api/base/filters.py

```diff
@@ -24,17 +24,30 @@
 def sort_multiple(fields):
     fields = list(fields)
     def sort_fn(a, b):
-        while fields:
-            field = fields.pop(0)
-            a_field = getattr(a, field)
-            b_field = getattr(b, field)
-            if a_field > b_field:
-                return 1
-            elif a_field < b_field:
-                return -1
+        for field in fields:
+            if field[0] == '-':
+                field = field[1:]
+                a_field = getattr(a, field)
+                b_field = getattr(b, field)
+                if a_field > b_field:
+                    return -1
+                elif a_field < b_field:
+                    return 1
+                elif a_field == b_field:
+                    return 1
+            else:
+                a_field = getattr(a, field)
+                b_field = getattr(b, field)
+                if a_field > b_field:
+                    return 1
+                elif a_field < b_field:
+                    return -1
+                elif a_field == b_field:
+                    return 1
```

Comment:
I'd like to see [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)er code here. We should be able to implement this without duplicating the `getattr` calls, nor the `>` and `<` comparisons (hint: set a flag to `1` or `-1` based on the `field[0]` comparison and multiply the `return` value by that).